data/lib/core/position.lua

```diff
@@ -50,14 +28,16 @@ function Position:moveUpstairs()
 	self.z = self.z - 1
 
 	local defaultPosition = self + Position.directionOffset[DIRECTION_SOUTH]
-	if not isWalkable(defaultPosition) then
+	local tileDefaultPosition = Tile(defaultPosition)
```

Comment:
But there is 2 Tile(pos) on this code.
Why it should be only there?
Also, I can't find any toTile() function.